.github/workflows/ci.yml

```diff
@@ -241,27 +241,20 @@ jobs:
       if: matrix.crate == 'zerocopy' && matrix.features == '--all-features' && matrix.toolchain == 'nightly'
 
   kani:
-    runs-on: ubuntu-latest
-    name: Run tests under Kani
-    needs: generate_cache
-    steps:
-      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
-      - name: Configure ZC_TOOLCHAIN environment variable
-        run: |
-          set -eo pipefail
-          echo "ZC_TOOLCHAIN=$(./cargo.sh --version nightly)" >> $GITHUB_ENV
-      - name: Install nigtly Rust toolchain (${{ env.ZC_TOOLCHAIN }})
-        uses: dtolnay/rust-toolchain@00b49be78f40fba4e87296b2ead62868750bdd83 # stable
-        with:
-          toolchain: ${{ env.ZC_TOOLCHAIN }}
-      - run: |
-          set -eo pipefail
-          cargo install --locked kani-verifier
-          cargo kani setup
-          RUSTFLAGS="$ZC_NIGHTLY_RUSTFLAGS" ./cargo.sh +nightly kani                   \
-            --package zerocopy --all-features --output-format terse --randomize-layout \
-            --memory-safety-checks --overflow-checks --undefined-function-checks       \
-            --unwinding-checks
+    on:
+      pull_request:
+      push:
+    jobs:
+      run-kani:
+        runs-on: ubuntu-latest
+        steps:
+          - name: 'Checkout your code.'
+            uses: actions/checkout@v3
```

Comment:
It's probably overkill to name this step, and you can also pin to a specific version of `checkout` like we do in other jobs:

```yaml
- uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
```

Reply:
Changing this right away